src/modules/components/dao/daoDataListItem/daoDataListItemStructure/daoDataListItemStructure.tsx

```diff
@@ -59,10 +54,6 @@ export const DaoDataListItemStructure: React.FC<IDaoDataListItemStructureProps>
                         <span className="capitalize">{network}</span>
                         <Icon icon={IconType.BLOCKCHAIN_BLOCKCHAIN} />
                     </div>
-                    <div className="flex items-center gap-2 text-sm md:text-base">
```

Comment:
As we're removing the plugin info, we can probably remove the wrapping div as well